frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -155,12 +155,7 @@ function FileManagerHistory:onShowHist()
 
     self:updateItemTable()
     self.hist_menu.close_callback = function()
-        -- Close it at next tick so it stays displayed
-        -- while a book is opening (avoids a transient
-        -- display of the underlying File Browser)
-        UIManager:nextTick(function()
-            UIManager:close(self.hist_menu)
-        end)
+        UIManager:close(self.hist_menu)
```

Comment:
(Not wanting to spend much time understand everything about this PR :) but this is no longer an issue ? In a few words, how is this taken care now ? Just by avoiding a setDirty when `not tearing_down` below?

Reply:
I'm not quite sure how/when it was actually an issue, but I can't reproduce it anymore.

Most likely because close_callback now runs *after* the Menu's action. But surely the useless refreshes that no longer happen probably help, too ;).